packages/gitbook/src/components/DocumentView/Table/RecordCard.tsx

```diff
@@ -28,9 +28,9 @@ export async function RecordCard(
         : null;
     const target = targetRef ? await context.resolveContentRef(targetRef) : null;
 
-    const coverIsLandscape =
+    const coverIsSquareOrPortrait =
         cover?.file?.dimensions &&
-        cover.file?.dimensions?.width / cover.file?.dimensions?.height > 1;
+        cover.file?.dimensions?.width / cover.file?.dimensions?.height < 1;
```

Comment:
Is square `< 1`?

Reply:
Ah damn yeah it should be

```suggestion
        cover.file?.dimensions?.width / cover.file?.dimensions?.height <= 1;
```